### PR TITLE
Consent revoke alert

### PR DESCRIPTION
**Pull request by ConstantinKV**
https://github.com/saltedge/sca-authenticator-android/issues/193

### CODE REVIEW

File: app/src/main/kotlin/com/saltedge/authenticator/features/main/SnackBuilder.kt [hunk 1]
```diff
@@ -28,31 +28,44 @@ import androidx.core.content.ContextCompat
...
+fun FragmentActivity.showWarningSnack(
     textResId: Int,
     snackBarDuration: Int = Snackbar.LENGTH_INDEFINITE,
     actionResId: Int? = null
-) = getSnackbarAnchorView()?.buildWarningSnack(textResId, snackBarDuration, actionResId)
+) = showWarningSnack(this.getString(textResId), snackBarDuration, actionResId)
 
-fun View.buildWarningSnack(textResId: Int, snackBarDuration: Int, actionResId: Int?) = buildSnackbar(
-    textResId = textResId,
+fun FragmentActivity.showWarningSnack(
```
Comment (mnewlive) [inline app/src/main/kotlin/com/saltedge/authenticator/features/main/SnackBuilder.kt:40]: this function shows nothing

File: app/src/main/kotlin/com/saltedge/authenticator/features/main/SnackBuilder.kt [hunk 2]
```diff
@@ -28,31 +28,44 @@ import androidx.core.content.ContextCompat
...
+    message: String,
+    snackBarDuration: Int = Snackbar.LENGTH_INDEFINITE,
+    actionResId: Int? = null
+) = getSnackbarAnchorView()?.showWarningSnack(message, snackBarDuration, actionResId)
+
+fun FragmentActivity.buildWarningSnack(
```
Comment (mnewlive) [inline app/src/main/kotlin/com/saltedge/authenticator/features/main/SnackBuilder.kt:46]: why do I need this function here?

File: app/src/main/kotlin/com/saltedge/authenticator/features/main/SnackBuilder.kt (hunk 1 above)
Comment (ConstantinKV) [reply app/src/main/kotlin/com/saltedge/authenticator/features/main/SnackBuilder.kt:40]: shows snack

File: app/src/main/kotlin/com/saltedge/authenticator/features/main/SnackBuilder.kt (hunk 2 above)
Comment (ConstantinKV) [reply app/src/main/kotlin/com/saltedge/authenticator/features/main/SnackBuilder.kt:46]: in one place i need just builder.

**Pull request merged**